seedlet-blocks/inc/block-styles.php

```diff
@@ -1,38 +0,0 @@
-<?php
-/**
- * Seedlet Theme: Block Styles
- *
- * @package Seedlet
- * @since 1.0.0
- */
-
-if ( ! function_exists( 'seedlet_blocks_register_block_styles' ) ) :
-
-	function seedlet_blocks_register_block_styles() {
-
-		if ( function_exists( 'register_block_style' ) ) {
-
-			/**
-			 * Register block styles
-			 */
-			register_block_style(
-				'core/latest-posts',
-				array(
-					'name'         => 'seedlet-alternating-grid',
-					'label'        => __( 'Alternating Grid', 'seedlet-blocks' ),
```

Comment:
Just confirming we want to remove these, based on @kjellr's reply in slack: p1642598004001100/1642590521.493900-slack-C029FM1EH

Reply:
Resolved.  Seedlet's stays for now.